projects/HSLink-OB/board/hslinkob/README_en.md

```
@@ -0,0 +1,111 @@
+
+## Pin Description
+
+(lab_hpm5301_evklite_board)=
+
+- UART Pin: modbus_rtu sample
+  - The UART0 used for debugger console or some functional testing using UART
+  - The UART3 is used for some functional testing using UART, such as MICROROS_UART, USB_CDC_ACM_UART, MODBUS_RTU etc.
+
+| Function  | Position | Remark   |
+| --------- | -------- | ------   |
+| UART3.TXD | J3[8]    |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)